driver-core/src/main/java/com/datastax/driver/core/AbstractGettableData.java

```diff
@@ -21,8 +21,6 @@
 import java.nio.ByteBuffer;
 import java.util.*;
 
-import com.google.common.collect.Lists;
```

Comment:
We allow ourselves to rebase and `push --force` working branches.
Anyway, that will be squashed before we merge to 2.0.